### ISSUE TITLE
NaNV error with Branes

### ISSUE THREAD

**Issue by rsmus7** [MarcBoule/Geodesics#7]
I have Branes in a patch and just one output gives me repeatedly a nanV value.
I need to load a new instance of Branes to get it working without the nanV.
The output is in the upper circle the first right from the middle output.
Would be great if you can fix this.

**Comment by MarcBoule** [comment]
Wow, that's quite weird, I was able to see this instantly, but I wonder why it was working before and that this is showing up just now. I'll take a look, thanks for reporting it.

**Comment by MarcBoule** [comment]
Very strange, I'm trying it now on another Windows machine, and I can't see the issue. I'll keep investigating, but if ever you have any additional context, please let me know. (Rack 2.5.2, Geodesics 2.3.2., Windows x64)

![image](https://github.com/user-attachments/assets/887b60f0-8d1a-4816-80d5-e5a1d2095ca5)

**Comment by rsmus7** [comment]
I'm really sorry, 
I couldn't reproduce the nanV bug today, 
I will try more tomorrow.

**Comment by MarcBoule** [comment]
I think I know the cause: there were some missing variable initializations (missing resets) in the code, and it's even surprising that this issue did not come up sooner, really. So I pushed a new commit that should fix it. I tried again on the same Windows machine on which I did confirm seeing the bug eariler today, I was not able to get the problem (right before implementing the fix). In any case, it's classic mistake, which would do exactly what we observed, and intermittent too, so it's very likely fixed now. I just don't know how soon we can push it to the library, I'll have to check with Pyer.  Cheers!
